main.go

```diff
@@ -198,15 +201,19 @@ func main() { // nolint:gocyclo
 			}
 		}()
 	}
-
+	webhookServer := webhook.NewServer(webhook.Options{
+		Port: 9443,
+	})
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
-		Port:                   9443,
+		Scheme: scheme,
+		Metrics: metricsserver.Options{
+			BindAddress: metricsAddr,
+		},
+		WebhookServer:          webhookServer,
 		HealthProbeBindAddress: probeAddr,
 		LeaderElection:         enableLeaderElection,
 		LeaderElectionID:       "dc07038f.toolchain.host.operator",
-		Namespace:              namespace,
+		Cache:                  cache.Options{DefaultNamespaces: map[string]cache.Config{namespace: {}}},
```

Comment:
I remember we discussed it and you explained how this new Cache configuration works but I already forgot :)
This configuration looks confusing (why there is default namespace as a map with a key but no value) so maybe you can try to add some comments explaining how it works? Or maybe a link to docs where it's explained. But feel free to leave it as is if it's not trivial.

Reply:
maybe this link? https://github.com/kubernetes-sigs/controller-runtime/blob/main/designs/cache_options.md

Reply:
fair enough, added the design doc link above in the comments in here - [b70c114](https://github.com/codeready-toolchain/host-operator/pull/1125/commits/b70c114afb1aa5d50df082bfc3798f8aa39f9f34)

Reply:
also removed webhook server, see [comment](https://github.com/codeready-toolchain/member-operator/pull/615/files#r1923385118)